packages/scripts/config/webpack.config.js

```diff
@@ -110,14 +116,17 @@ const cssLoaders = [
 	},
 ];
 
-const config = {
+/** @type {webpack.Configuration} */
+const baseConfig = {
 	mode,
 	target,
-	entry: getWebpackEntryPoints,
 	output: {
 		filename: '[name].js',
 		path: resolve( process.cwd(), 'build' ),
 	},
+	experiments: {
+		outputModule: true,
+	},
```

Comment:
Why is this on the `baseConfig` instead of `moduleConfig`? Couldn't it affect the plugins extending the Webpack config like this, or is it innocuous?

```js
const defaultConfig = require("@wordpress/scripts/config/webpack.config");

module.exports = {
  ...defaultConfig,
  // my extra config...
};
```

Reply:
As far as I can tell, by itself this doesn't change anything about how webpack works. It enables the module output experiment, but that's controlled by other options like `output.module`.

This seemed like a harmless, simple change, but there's no real justification to do it here. I'll move it to the module config.